contextvh/src/main/java/contextvh/ContextEntity.java

```diff
@@ -30,6 +36,7 @@
 	 */
 	public ContextEntity(final EntityListener env, final String intendedStakeholder, final Integer slotID) {
 		super(env, intendedStakeholder, slotID);
+		customActions.addAction(perceptsFilter);
```

Comment:
@spiderbiggen  This works with relevantarea's because it's only necessary when the action is called (and thus given as parameter).FilterPercepts is also needed when other percepts are send and the action hasn't been called for a while.

See our [internal pr](https://github.com/Denpeer/tygron/pull/23) for more information